core/src/test/java/io/parsingdata/metal/ReferenceValueExpressionSemanticsTest.java

```diff
@@ -101,4 +108,11 @@ private static Token refMatch(final Expression pred) {
                 def("b", con(1), pred));
     }
 
+    private static Token limitedSum(final int limit) {
+        return
+            seq(rep(def("a", con(1), not(eq(con(0))))),
+                def("zero", con(1), eq(con(0))),
+                def("sum", con(1), eq(fold(new NameRef("a", con(limit)), Shorthand::add))));
```

Comment:
It's a remnant from before there was a shorthand which included the `limit` argument :) I'll fix it.